app/models/allocation_history_item.rb

```diff
@@ -0,0 +1,19 @@
+# View Object for an allocation history item
+
```

Comment:
Should we be using the presenter pattern now it's been added? (I'm ambivalent)

Reply:
I think we should as it would be good to use one 'thing', rather than have separate concepts for essentially the same thing.